README.md

```diff
@@ -1 +1,64 @@
-# Bookstore
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Built With
+
+- React
+- JavaScript
+- Webpack
+- Gitflow
+- JSX
+
+## Live Demo
+
+- Watch the live demo with [Netlify](https://math-magicians-jsotomayor.netlify.app/)
+
+- Watch the live demo with [Heroku](https://math-magicians-jsotomayor.herokuapp.com/)
```

Comment:
- [ ] OPTIONAL
Kindly update your _Live demo_  links to reflect the current project. This will make your ReadMe appear more professional